File: TASK-03/javascript/subtask3.js
const readline = require("readline").createInterface({
    input: process.stdin,
    output: process.stdout,
});

readline.question("Enter an odd number: ", n => {
    n = parseInt(n);

    // Upper part
    for (let i = 0; i <= Math.floor(n / 2); i++) {
        console.log(" ".repeat(Math.floor(n / 2) - i) + "*".repeat(2 * i + 1));
    }

    // Lower part
    for (let i = Math.floor(n / 2) - 1; i >= 0; i--) {
        console.log(" ".repeat(Math.floor(n / 2) - i) + "*".repeat(2 * i + 1));
    }

    readline.close();
});

File: TASK-03/java/subtask3.java
import java.util.Scanner;

public class subtask3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter an odd number: ");
        int n = scanner.nextInt();

        // Upper part
        for (int i = 0; i <= n / 2; i++) {
            System.out.println(" ".repeat(n / 2 - i) + "*".repeat(2 * i + 1));
        }

        // Lower part
        for (int i = n / 2 - 1; i >= 0; i--) {
            System.out.println(" ".repeat(n / 2 - i) + "*".repeat(2 * i + 1));
       }
    }
}
